pyproject.toml

```diff
@@ -39,12 +39,13 @@ pytest = "*"
 pyyaml = "*"
 pydocstyle = "*"
 m2r2 = "*"
-sphinx = "*"
 sphinx-rtd-theme = "*"
 toml = "*"
 yamllint = "*"
 mypy = "^0.961"
 black = "*"
+napalm = "^4.0.0"
```

Comment:
I think this should also be in `[tool.poetry.extras]`. dev-dependencies are for your testing strategy, but extras allows you to do something like `pip install netutils[napalm]`

Reply:
```
▶ poetry add -E netlibs napalm
The following packages are already present in the pyproject.toml and will be skipped:

  • napalm

If you want to update it to the latest compatible version, you can use `poetry update package`.
If you prefer to upgrade it to the latest available version, you can use `poetry add package@latest`.

Nothing to add.
```

Reply:
@itdependsnetworks better way to do this?

Reply:
Do you mind pinging ask-python internally? I haven't set this up personally before. Phillip may have done it on jsonschema as an example.